wpilibj/src/athena/java/edu/wpi/first/wpilibj/Ultrasonic.java

```diff
@@ -288,15 +288,8 @@ public void ping() {
     setAutomaticMode(false); // turn off automatic round robin if pinging
     // single sensor
     m_counter.reset(); // reset the counter to zero (invalid data now)
-    m_pingChannel.pulse(m_pingChannel.m_channel, (float) kPingTime); // do
-    // the
-    // ping
-    // to
-    // start
-    // getting
-    // a
-    // single
-    // range
+    m_pingChannel.pulse(m_pingChannel.getChannel(), (float) kPingTime); // do
+    // the ping to start getting a single range
```

Comment:
Could reformat this a bit better by putting the entire comment on the line before the code (instead of just having `// do` on the code line).